from0k2bp.tex

```diff
@@ -69,32 +69,31 @@
 might find this investigation at least partly as interesting to read as
 I found it to construct!
 
-\textbf{Knowledge prerequisites}: you're not assumed to know anything
-about elliptic curves, except that points add, can be expressed as
-scalar multiples, i.e. the standard public-private key relationship like
-$P=kG$, with $x$ the scalar value which is the private key, and understand why it
+\textbf{Knowledge prerequisites}: You are not assumed to know anything
+about elliptic curves (EC) except that EC points can be added and also EC points 
+can be expressed as scalar multiples, i.e. the standard public-private key relationship like
+$P=kG$ where $k$ is a scalar value which represents the private key, and understand why it
 makes sense to say something like $aP = axG$, and that if $P-Q=H$ and $Q=rsG$ then the private key
 of $H$ would be $(x-rs)$. No deeper mathematics about the \emph{construction} of
 elliptic curves will be relevant here. If you've already taken the time
 to understand e.g. ECDSA signatures or Schnorr signatures, you'll find
 nothing here to trouble you.
 
-Similar comments apply to hash functions (which play only a very minor
-role in most of this document).
+Similar comments apply to hash functions which play only a very minor
```

Comment:
This is not an improvement of the English - if you remove the parentheses, then you have to add a comma before 'which' (it's literally a semantically different sentence without the comma, albeit the difference is a bit subtle; are you referring only to hash functions which play a minor role (and not hash functions which play a big role), or are you saying that hash functions in general play a minor role? Obviously the latter is correct, and so the comma needs to be included if you don't use parentheses.

Reply:
I see. Reverted.